js/recipe/introduction.js

```diff
@@ -0,0 +1,17 @@
+import { recipeParentElement } from "../constants.js"
+import { render } from "../htmlRenderer.js"
+import * as els from "../htmlRenderer.js"
+
+export function introduction(recipe) {
+  const firstParagraphElement = els.p({innerHTML: recipe.description[0]});
+  const firstParagraphDiv = els.div({className: "first-paragraph", children: [firstParagraphElement]});
+  const latterDescriptionParagraphs = recipe.description.slice(1);
+
+  const latterParagraphElements = latterDescriptionParagraphs.map(paragraph => {
+    return els.p({innerHTML: paragraph});
+  });
```

Comment:
This can be written as:
```js
const latterParagraphElements = latterDescriptionParagraphs.map(paragraph => els.p({innerHTML: paragraph}));
```

where we have an implicit return value from the map function.

Reply:
Yep, I figured out that adding the additional curly braces meant that the return was required. I'm still learning the basics of es6 syntax!